lib/network/modules/components/nodes/shapes/Image.js

```diff
@@ -58,8 +58,19 @@ class Image extends CircleImageBase {
     ctx.save();
     this.switchImages(selected);
     this.resize();
-    this.left = x - this.width / 2;
-    this.top = y - this.height / 2;
+
+    var labelX = x,
+      labelY = y;
```

Comment:
Are the changes in this file any different to the changes in CircularImage.js?

Reply:
Image overrides `draw` from CircularImage to add some extra code (and pretty much copy the rest of the existing code). I'm just mimicking what's already done.  I don't think it would be worth it to add yet another function call for just these few lines.

Reply:
The code quality here is not particularly good but I understand if you don't want to dive too deep into this.